prerequisites/software.rst

```diff
@@ -69,18 +70,26 @@ The following reverse proxies are supported:
 5. Elasticsearch (optional)
 ===========================
 
-.. note:: Package install will insist on installing elasticsearch, you can break dependencies during install if needed.
+For excellent search performance we use Elasticsearch. 
+While package install will insist on installing Elasticsearch, you can break those dependencies if needed.
 
 .. warning:: Please note that if you do not install and use Elasticsearch, the search will be very limited!
    We recommend using Elasticsearch, as it will boost the usuage of Zammad greatly!
 
-For excellent search performance we use Elasticsearch.
-The following Elasticsearch versions are supported:
 
-* Elasticsearch 5.5 with ``mapper-attachments`` plugin
-* Elasticsearch 5.6, 6.x & 7.x with ``ingest-attachment`` plugin
+The Elasticsearch support differs on the Elasticsearch and Zammad version you're using:
```

Comment:
I think it's safe to remove this line—since we have the table below, the reader doesn't gain anything by having this sentence here.

Reply:
![](https://media1.tenor.com/images/80ad87633e83ed66baf78d6608626b26/tenor.gif)